### PR TITLE
Adds basic xenoarch & exploration equipment to research robot module

### PR DESCRIPTION
**Pull request by atlantiscze**
- Also removes commented-out sources of construction module which is long unused.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules/station.dm [hunk 1]
```diff
@@ -789,6 +741,10 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/shockpaddles/robot/jumper(src)
 	src.modules += new /obj/item/weapon/melee/baton/slime/robot(src)
 	src.modules += new /obj/item/weapon/gun/energy/taser/xeno/robot(src)
+	src.modules += new /obj/item/device/xenoarch_multi_tool(src)
+	src.modules += new /obj/item/weapon/pickaxe/excavationdrill(src)
+	src.modules += new /obj/item/device/cataloguer(src)
+
```
Comment (TheFurryFeline) [inline code/modules/mob/living/silicon/robot/robot_modules/station.dm:747]: What about a depth analysis scanner for the xenoarch portion?

Comment (atlantiscze) [reply code/modules/mob/living/silicon/robot/robot_modules/station.dm:747]: Xenoarch multitool contains a depth analysis scanner (as well as alden-saraspova counter and measuring tape)

**Comment by atlantiscze** [comment]
No idea why the Travis build is failing, i didn't touch that part of code at all. Could someone with repo access perhaps try restarting it?

File: code/modules/mob/living/silicon/robot/robot_modules/station.dm (hunk 1 above)
Comment (TheFurryFeline) [reply code/modules/mob/living/silicon/robot/robot_modules/station.dm:747]: Oh, nice. Didn't realise that.

**Comment by Atermonera** [comment]
Travis has a tendency to fall over, if I'm looking at PRs and I notice that it failed on one I'll usually take a look at it. Although that is subject to me looking at PRs.

**Pull request merged**